paddle/fluid/inference/api/paddle_inference_api.h

```diff
@@ -259,11 +259,20 @@ struct AnalysisConfig : public NativeConfig {
     kExclude   // Specify the disabled passes in `ir_passes`.
   };
 
+  void SetIncludeMode() {
+    ir_mode = IrPassMode::kInclude;
+    ir_passes = {"infer_clean_graph_pass"};
+    ir_mkldnn_passes = {"infer_clean_graph_pass"};
+  }
```

Comment:
why should add `infer_clean_graph_pass`?

Reply:
It seems this pass always has to run at the beginning of all fuse passes. Otherwise, the whole optimization fails. Thus this pass can be added as the first one by default.

Reply:
Got it, you can add this comment in the codes.